src/PaymentGateways/Stripe/Controllers/NewStripeAccountOnBoardingController.php

```diff
@@ -38,6 +42,15 @@ public function __invoke()
             return;
         }
 
+        $isDonationFormPage = isset($_GET['give_tab']) && $_GET['give_tab'] !== 'stripe_form_account_options';
+        $isSettingPage = isset($_GET['tab'], $_GET['tab'])
+                         && Give_Admin_Settings::is_setting_page('gateways', 'stripe-settings');
```

Comment:
These params: `give_tab`, `tab` seem a little too generic - I think we should validate more of the url structure to make sure its actually on a donation form edit screen or givewp gateway settings screen.

It would also be nice if you can write some tests for this controller 😄